### PR TITLE
Replace BNDW with VAGF

### PR DESCRIPTION
**Pull request by zupo**
VAGF is EUR-hedged and auto-invests dividends. This is much better for most EU citizens.

### CODE REVIEW

File: 5_People/personal-finances.md [hunk 1]
```diff
@@ -92,8 +92,8 @@ We'll be buying whole-market index funds for stocks and bonds.
 
 We're looking to invest in the lowest cost funds by a large and reputable fund. Niteo, Dejan, and Nejc all invest in [Vanguard's](https://en.wikipedia.org/wiki/The_Vanguard_Group) funds:
 
-- VWRA for the global stock market (listed as VWCE in EUR)
-- BNDW for the global bond market
+- VWCE for the global stock market (listed as VWCE in EUR)
```
Comment (dmurko) [inline 5_People/personal-finances.md:95]:
```suggestion
- VWCE (EUR) or VWRA (USD) for the global stock market
```

File: 5_People/personal-finances.md [hunk 2]
```diff
@@ -92,8 +92,8 @@ We'll be buying whole-market index funds for stocks and bonds.
...
+- VAGF for the global bond market
```
Comment (dmurko) [inline 5_People/personal-finances.md:96]:
```suggestion
- VAGF (EUR) or BNDW (USD) for the global bond market
```

**Review by dmurko — COMMENTED** [review]
Maybe like this then?

**Comment by zupo** [comment]
Better?

**Review by dmurko — APPROVED** [review]

**Pull request merged**